### PR TITLE
Updated PHPUnit constraint to support both v5 and v6

### PR DESCRIPTION
**Pull request by MarioBlazek**

### CODE REVIEW

**Comment by emodric** [comment]
:+1:

**Comment by andrerom** [comment]
Looking into the failure in #64

**Comment by andrerom** [comment]
@MarioBlazek you can rebase on master now, it should pass unless this introduces issues.

**Comment by emodric** [comment]
@andrerom @MarioBlazek is on vacation, but if you want and if it's enabled, you should be able to rebase the branch directly in his fork since you're a maintainer.

**Comment by MarioBlazek** [comment]
@andrerom rebase done.

**Comment by andrerom** [comment]
It fails with `Fatal error: Class 'PHPUnit_Framework_Assert' not found (Behat\Testwork\Call\Exception\FatalThrowableError)`: https://travis-ci.org/ezsystems/BehatBundle/builds/250407482

**Comment by emodric** [comment]
@andrerom I can't see where it comes from, but it can be solved with something like this: https://github.com/netgen/ezpublish-kernel/pull/55/files

**Comment by MarioBlazek** [comment]
Class alias helped, thanks @emodric.

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "require": {
         "php": "^5.6|^7.0",
         "ezsystems/ezpublish-kernel": "*",
-        "phpunit/phpunit": "~5.0"
+        "phpunit/phpunit": "^5.0|^6.0"
```
Comment (andrerom) [inline composer.json]: With the changes on the other repos I guess this should be `^5.7|^6.0` right?

Besides that, maybe add a comment before the alias stating it is temporary until all packages using this bundle supports phpunit 6.

**Comment by MarioBlazek** [comment]
Updated as suggested by @andrerom.

**Comment by emodric** [comment]
@andrerom Is tag possible any time soon? :)

**Comment by andrerom** [comment]
@emodric You mean like this one https://github.com/ezsystems/BehatBundle/releases/tag/v6.5.1 ;)

**Comment by emodric** [comment]
Thanks :)

**Pull request merged**